src/debug.rs

```diff
@@ -0,0 +1,52 @@
+//! Interacting with debugging agent
```

Comment:
Could you add an example of how may want to use this? Since the Exception enum has fixed codes, I suppose this is based on some sort of standard.